{"name": "Process a phone order with Save and then Pay", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\n +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\n +    where order = <logic.takeout_logic.Calculations object at 0x0000019716367160>.order\n +      where <logic.takeout_logic.Calculations object at 0x0000019716367160> = <test_cash.test_phoneorder_cash.TestPhoneOrderCash object at 0x00000197162CA6E0>.phone", "trace": "self = <test_cash.test_phoneorder_cash.TestPhoneOrderCash object at 0x00000197162CA6E0>\n\n    @allure.story(\"Phone Order Save and Pay\")\n    @allure.title(\"Process a phone order with Save and then Pay\")\n    def test_phoneorder_save_pay(self):\n    \n>       assert self.phone.order(\"Phone Order\") == True\nE       AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\nE        +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\nE        +    where order = <logic.takeout_logic.Calculations object at 0x0000019716367160>.order\nE        +      where <logic.takeout_logic.Calculations object at 0x0000019716367160> = <test_cash.test_phoneorder_cash.TestPhoneOrderCash object at 0x00000197162CA6E0>.phone\n\nAutomation-logic\\suits\\test_orderflows\\test_cash\\test_phoneorder_cash.py:31: AssertionError"}, "start": 1742910250241, "stop": 1742910261069, "uuid": "30037524-bfac-4598-89d7-1444c2f7dea6", "historyId": "545ae58e27671a422d1877353cf39130", "testCaseId": "545ae58e27671a422d1877353cf39130", "fullName": "suits.test_orderflows.test_cash.test_phoneorder_cash.TestPhoneOrderCash#test_phoneorder_save_pay", "labels": [{"name": "epic", "value": "Orderflow"}, {"name": "story", "value": "Phone Order Save and Pay"}, {"name": "feature", "value": "Phone order in Cash"}, {"name": "parentSuite", "value": "suits.test_orderflows.test_cash"}, {"name": "suite", "value": "test_phoneorder_cash"}, {"name": "subSuite", "value": "TestPhoneOrderCash"}, {"name": "host", "value": "LAPTOP-KKMKONCC"}, {"name": "thread", "value": "9204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "suits.test_orderflows.test_cash.test_phoneorder_cash"}]}